tidy3d/components/simulation.py

```diff
@@ -3670,3 +3823,15 @@ def subsection(
         )
 
         return new_sim
+
+    def suggest_mesh_overrides(self) -> List[MeshOverrideStructure]:
```

Comment:
In the future, we might want to suggest for other structures, e.g. bulk PEC. so perhaps we should change the function signature to `(self, **kwargs)` to allow for extension?